src/framework/Elsa.Studio.Core/SyntaxProviders/CSharpSyntaxProvider.cs

```diff
@@ -0,0 +1,20 @@
+using Elsa.Api.Client.Contracts;
+using Elsa.Api.Client.Expressions;
+using Elsa.Studio.Abstractions;
+
+namespace Elsa.Studio.SyntaxProviders;
+
+/// <summary>
+/// A provider of a JavaScript syntax that is used by Monaco.
```

Comment:
```suggestion
/// A provider of a C# syntax that is used by Monaco.
```

Reply:
I didn't know about this GitHub feature - very handy! 😃